templates/pipelines/common-techdocs-publish.yaml

```diff
@@ -45,6 +45,7 @@ stages:
                  -PSHelperDirectory ${{ variables.PSHelperDirectory }}
                 pwsh: true
                 failOnStderr: true
+                warningPreference: silentlyContinue
```

Comment:
why is this silentContinue ?

Reply:
it was used to avoid npm install warnings. It is now mitigated using --loglevel=error. Hence removed warningPreference